index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt('Ingrese su nombre:';)
+alert('Hola, '+ nombreUsuario} + '!');
```

Comment:
Hay un typo en esta línea con un corchete.
```suggestion
alert('Hola, '+ nombreUsuario + '!');
```